.gitignore

```diff
@@ -14,7 +14,6 @@ dist/
 **/db.tmp
 dbFiles/*
 *.jar
-ui/server/src/main/resources/web
```

Comment:
Please check if it isn't used in dist preparing. If so, split web to web (sources) and web-generated (generated sources - ignored in git).

Reply:
Frontend stuff is copied by webpack:

```
  output: {
    //by default we use default webpack value, but we want to be able to override it for building frontend via sbt
    path: process.env.OUTPUT_PATH ? path.join(process.env.OUTPUT_PATH, "classes", "web", "static") : path.join(process.cwd(), "dist"),
    filename: "[name].js",
    //see config.js
    publicPath: isProd ? "__publicPath__/static/" : "/static/",
  },
```
It is copied to target.